src/main/java/com/ibm/cloudant/kafka/connect/CommonConnectorConfig.java

```diff
@@ -0,0 +1,125 @@
+    public static final String CLOUDANT_USERNAME_DEFAULT = null;
+    public static final String CLOUDANT_PASSWORD_DEFAULT = null;
+    public static final String CLOUDANT_LAST_SEQ_NUM_DEFAULT = "0";
+
+    public static ConfigDef baseConfigDef() {
+
+        return new ConfigDef()
+
+                // Cloudant URL
+                .define(InterfaceConst.URL,
+                        Type.STRING,
+                        ConfigDef.NO_DEFAULT_VALUE,
+                        new UrlValidator(),
+                        Importance.HIGH,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_URL_DOC),
+                        DATABASE_GROUP,
+                        1,
+                        Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_URL_DISP))
+                // Cloudant DB
+                .define(InterfaceConst.DB,
+                        Type.STRING,
+                        Importance.HIGH,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_DB_DOC),
+                        DATABASE_GROUP,
+                        1,
+                        Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_DB_DISP))
+                // Cloudant Username
+                .define(InterfaceConst.USER_NAME,
+                        Type.STRING,
+                        CLOUDANT_USERNAME_DEFAULT,
+                        Importance.HIGH,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_USR_DOC),
+                        DATABASE_GROUP,
+                        1,
+                        Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_USR_DISP))
+                // Cloudant Password
+                .define(InterfaceConst.PASSWORD,
+                        Type.PASSWORD,
+                        CLOUDANT_PASSWORD_DEFAULT,
+                        Importance.HIGH,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_PWD_DOC),
+                        DATABASE_GROUP,
+                        1,
+                        Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_PWD_DISP))
+                // Cloudant auth type
+                .define(InterfaceConst.AUTH_TYPE,
+                        Type.STRING,
+                        CLOUDANT_AUTH_TYPE_DEFAULT,
+                        new ListRecommender(CLOUDANT_AUTH_TYPE_BASIC),
+                        Importance.LOW,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_AUTH_TYPE_DOC),
+                        DATABASE_GROUP,
+                        1,
+                        Width.LONG,
+                        ResourceBundleUtil.get(MessageKey.CLOUDANT_CONNECTION_AUTH_TYPE_DISP),
+                        new ListRecommender(CLOUDANT_AUTH_TYPE_BASIC))
```

Comment:
I think this is ok as a "placeholder" but I think we'll want to increase this from "low" and change the default.